samples/cpp/benchmark_app/main.cpp

```diff
@@ -804,7 +811,17 @@ int main(int argc, char* argv[]) {
         // ----------------------------------------
         next_step();
 
+        auto create_requests_start_time = Time::now();
         InferRequestsQueue inferRequestsQueue(compiledModel, nireq, app_inputs_info.size(), FLAGS_pcseq);
+        auto create_requests_duration_ms = get_duration_ms_till_now(create_requests_start_time);
+        slog::info << "Creating " << nireq << " infer requests took " << double_to_string(create_requests_duration_ms)
+                   << " ms" << slog::endl;
+
+        if (statistics) {
+            statistics->add_parameters(
+                StatisticsReport::Category::EXECUTION_RESULTS,
+                {StatisticsVariant("create infer requests time (ms)", "create_requests", create_requests_duration_ms)});
```

Comment:
Where did this metric requirement come from?

Reply:
Regarding the first comment - why do we list compiledModel's properties multiple times (for every device)? The device variable is only used then in the log output. Should I extract the supported properties logging loop outside of the device logging one?
Regarding the second comment - this is an alignment with Python report output, in step 9 this metric is added therefore I decided to implement it in C++ benchmark_app as well.

Reply:
```
for (const auto& device : devices) {
            auto supported_properties = compiledModel.get_property(ov::supported_properties);
            slog::info << "Device: " << device << slog::endl;
            for (const auto& cfg : supported_properties) {
                try {
                    if (cfg == ov::supported_properties)
                        continue;
                    auto prop = compiledModel.get_property(cfg);
                    slog::info << "  { " << cfg << " , " << prop.as<std::string>() << " }" << slog::endl;
                } catch (const ov::Exception&) {
                }
            }
        }
 ```
 @nkogteva This is the current loop in the master branch - it does not appear as if it's using the 'device' variable when requesting the list of supported properties. Shouldn't the line `compiledModel.get_property(ov::supported_properties);` be replaced by `core.get_property(device, ov::supported_properties);` in that case, and leave the `compiledModel.get_property(cfg)` as it is?

Reply:
Oki, removed. Updated READMEs for both benchmarks as well.